docs/modules/vertiq_4006.rst

```diff
@@ -0,0 +1,21 @@
+********************************
+Vertiq 4006 | Speed Module
+********************************
+
+.. sidebar:: Vertiq 4006
+
+    .. image:: ../_static/Vertiq4006.jpg
```

Comment:
This doesn't work when its online, I built it with read-the-docs and checked. You need the capitalization of the extensions to match, so this should be .JPG since you have .JPG in your static file

Reply:
Should I rename the static file to lowercase .jpg for consistency?